lib/membrane_aac_format/aac.ex

```diff
@@ -3,60 +3,65 @@ defmodule Membrane.AAC do
   Capabilities for [Advanced Audio Codec](https://wiki.multimedia.cx/index.php/Understanding_AAC).
   """
 
-  @type profile_t :: :main | :LC | :SSR | :LTP | :HE | :HEv2
-  @type mpeg_version_t :: 2 | 4
-  @type samples_per_frame_t :: 1024 | 960
+  @type profile :: :main | :LC | :SSR | :LTP | :HE | :HEv2
+  @type mpeg_version :: 2 | 4
+  @type samples_per_frame :: 1024 | 960
 
   @typedoc """
   Indicates whether stream contains AAC frames only or are they encapsulated
   in [ADTS](https://wiki.multimedia.cx/index.php/ADTS)
   """
-  @type encapsulation_t :: :none | :ADTS
+  @type encapsulation :: :none | :ADTS
 
   @typedoc """
   Identifiers of [MPEG Audio Object Types](https://wiki.multimedia.cx/index.php/MPEG-4_Audio#Audio_Object_Types)
   """
-  @type audio_object_type_id_t :: 1..5 | 29
+  @type audio_object_type_id :: 1..5 | 29
 
   @typedoc """
   Identifiers of [MPEG Audio sampling frequencies](https://wiki.multimedia.cx/index.php/MPEG-4_Audio#Sampling_Frequencies)
   """
-  @type sampling_frequency_id_t :: 0..12 | 15
+  @type sampling_frequency_id :: 0..12 | 15
 
   @typedoc """
   Identifiers of [MPEG Audio channel configurations](https://wiki.multimedia.cx/index.php/MPEG-4_Audio#Channel_Configurations)
   """
-  @type channel_config_id_t :: 0..7
+  @type channel_config_id :: 0..7
 
   @typedoc """
   AAC frame length identifiers.
 
   `0` indicates 1024 samples/frame and `1` - 960 samples/frame.
   """
-  @type frame_length_id_t :: 0 | 1
+  @type frame_length_id :: 0 | 1
+
+  @typedoc """
+  Contains the configuration data of the audio stream. The options are:
+  * Elementary Stream Descriptor, `:esds` - ISO/IEC 14496-14
+  * Audio Specific Config, `:audio_specific_config` - ISO/IEC 14496-3
+  If the data is already parsed it can be set to nil.
+  """
+  @type config :: {:esds | :audio_specific_config, binary()}
```

Comment:
Perhaps it would be better to do it that way:
```suggestion
  If the data is already parsed it can be set to nil.
  """
  @type config :: {:esds | :audio_specific_config, binary()} | nil
``` 
rather than specifying `config: config | nil` in type `t`, 
since it's that typedoc, in which we are describing`nil` semantics?